### PR TITLE
docs: add minimal resource guidance for provisioning clusters on publ…

### PR DESCRIPTION
**Pull request by lennessyy**
## Describe the Change

This PR adds a sentence that clarifies the minimal resources required to deploy a cluster. 

## Review Changes

💻 [Add Preview URL]()

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/DOC-839)

### CODE REVIEW

File: docs/docs-content/clusters/public-cloud/aws/create-cluster.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Use the following steps to provision a new AWS cluster:
   
 
 
-10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool.
+10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool. The minimum number of CPUs and amount of memory depends on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools.
```
Comment (lennessyy) [inline docs/docs-content/clusters/public-cloud/aws/create-cluster.md:103]:
```suggestion
10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools. 
```

File: docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ When you have provided all the cluster configuration details to the wizard, clic
 
 <br />
 
-7. Configure the master and worker node pools. A master and a worker node pool are configured by default. To learn more about the configuration options, review the [Node Pool](../../cluster-management/node-pool.md) documentation page.
+7. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depends on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools. To learn more about the configuration options, review the [Node Pool](../../cluster-management/node-pool.md) documentation page.
```
Comment (lennessyy) [inline docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md:116]:
```suggestion
7. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools. To learn more about the configuration options, review the [Node Pool](../../cluster-management/node-pool.md) documentation page.

```

File: docs/docs-content/clusters/public-cloud/gcp/create-gcp-gke-cluster.md [hunk 3]
```diff
@@ -74,7 +74,7 @@ Ensure the following requirements are met before you attempt to deploy a cluster
   |**Region**|Choose the desired GCP region in which to deploy the cluster.|
 
 
-11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the worker node pool.
+11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the worker node pool. The minimum number of CPUs and amount of memory depends on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools.
```
Comment (lennessyy) [inline docs/docs-content/clusters/public-cloud/gcp/create-gcp-gke-cluster.md:77]:
```suggestion
11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the worker node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools. 
```

File: docs/docs-content/clusters/public-cloud/gcp/create-gcp-iaas-cluster.md [hunk 4]
```diff
@@ -87,7 +87,7 @@ Ensure the following requirements are met before you attempt to deploy a cluster
 
   
 
-11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the master and worker node pools. A master and a worker node pool are configured by default.  
+11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depends on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools.
```
Comment (lennessyy) [inline docs/docs-content/clusters/public-cloud/gcp/create-gcp-iaas-cluster.md:90]:
```suggestion
11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools.  
```

File: docs/docs-content/clusters/public-cloud/aws/create-cluster.md [hunk 5]
```diff
@@ -100,7 +100,7 @@ Use the following steps to provision a new AWS cluster:
   
 
 
-10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool.
+10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools.
```
Comment (lennessyy) [inline docs/docs-content/clusters/public-cloud/aws/create-cluster.md:103]:
```suggestion
10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory both in the master pool and across all worker pools. 
```

File: docs/docs-content/clusters/public-cloud/gcp/create-gcp-gke-cluster.md [hunk 6]
```diff
@@ -74,7 +74,7 @@ Ensure the following requirements are met before you attempt to deploy a cluster
   |**Region**|Choose the desired GCP region in which to deploy the cluster.|
 
 
-11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the worker node pool.
+11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the worker node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools.
```
Comment (lennessyy) [inline docs/docs-content/clusters/public-cloud/gcp/create-gcp-gke-cluster.md:77]:
```suggestion
11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the worker node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory both in the master pool and across all worker pools. 
```

File: docs/docs-content/clusters/public-cloud/gcp/create-gcp-iaas-cluster.md [hunk 7]
```diff
@@ -87,7 +87,7 @@ Ensure the following requirements are met before you attempt to deploy a cluster
 
   
 
-11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the master and worker node pools. A master and a worker node pool are configured by default.  
+11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools.
```
Comment (lennessyy) [inline docs/docs-content/clusters/public-cloud/gcp/create-gcp-iaas-cluster.md:90]:
```suggestion
11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory both in the master pool and across all worker pools.  
```

File: docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md [hunk 8]
```diff
@@ -113,7 +113,8 @@ When you have provided all the cluster configuration details to the wizard, clic
 
 <br />
 
-7. Configure the master and worker node pools. A master and a worker node pool are configured by default. To learn more about the configuration options, review the [Node Pool](../../cluster-management/node-pool.md) documentation page.
+7. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in the master pool and across all worker pools. To learn more about the configuration options, review the [Node Pool](../../cluster-management/node-pool.md) documentation page.
```
Comment (lennessyy) [inline docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md:116]:
```suggestion
7. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory both in the master pool and across all worker pools. To learn more about the configuration options, review the [Node Pool](../../cluster-management/node-pool.md) documentation page.

```

File: docs/docs-content/clusters/public-cloud/aws/create-cluster.md [hunk 9]
```diff
@@ -100,7 +100,7 @@ Use the following steps to provision a new AWS cluster:
   
 
 
-10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool.
+10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory both in the master pool and across all worker pools.
```
Comment (ritawatson) [inline docs/docs-content/clusters/public-cloud/aws/create-cluster.md:103]:
```suggestion
10. Configure the master and worker node pools. A master and a worker node pool are configured by default. This is the section where you can specify the availability zones (AZ), instance types, [instance cost type](architecture.md#spot-instances), disk size, and the number of nodes. Click on **Next** after you have completed configuring the node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in both the master pool and across all worker pools. 
```

File: docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md [hunk 10]
```diff
@@ -113,7 +113,9 @@ When you have provided all the cluster configuration details to the wizard, clic
 
 <br />
 
-7. Configure the master and worker node pools. A master and a worker node pool are configured by default. To learn more about the configuration options, review the [Node Pool](../../cluster-management/node-pool.md) documentation page.
+7. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory both in the master pool and across all worker pools. To learn more about the configuration options, review the [Node Pool](../../cluster-management/node-pool.md) documentation page.
```
Comment (ritawatson) [inline docs/docs-content/clusters/public-cloud/azure/create-azure-cluster.md:116]:
```suggestion
7. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in both the master pool and across all worker pools. To learn more about the configuration options, review the [Node Pool](../../cluster-management/node-pool.md) documentation page.

```

File: docs/docs-content/clusters/public-cloud/gcp/create-gcp-iaas-cluster.md [hunk 11]
```diff
@@ -87,7 +87,7 @@ Ensure the following requirements are met before you attempt to deploy a cluster
 
   
 
-11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the master and worker node pools. A master and a worker node pool are configured by default.  
+11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory both in the master pool and across all worker pools.
```
Comment (ritawatson) [inline docs/docs-content/clusters/public-cloud/gcp/create-gcp-iaas-cluster.md:90]:
```suggestion
11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the master and worker node pools. A master and a worker node pool are configured by default. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in both the master pool and across all worker pools.  
```

File: docs/docs-content/clusters/public-cloud/gcp/create-gcp-gke-cluster.md [hunk 12]
```diff
@@ -74,7 +74,7 @@ Ensure the following requirements are met before you attempt to deploy a cluster
   |**Region**|Choose the desired GCP region in which to deploy the cluster.|
 
 
-11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the worker node pool.
+11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the worker node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory both in the master pool and across all worker pools.
```
Comment (ritawatson) [inline docs/docs-content/clusters/public-cloud/gcp/create-gcp-gke-cluster.md:77]:
```suggestion
11. The Node configuration page is where you can specify the availability zones (AZ), instance types, disk size, and the number of nodes. Configure the worker node pool. The minimum number of CPUs and amount of memory depend on your cluster profile, but in general you need at least 4 CPUs and 4 GB of memory in both the master pool and across all worker pools. 
```

File: docs/docs-content/clusters/public-cloud/aws/create-cluster.md (hunk 9 above)
Comment (ritawatson) [reply docs/docs-content/clusters/public-cloud/aws/create-cluster.md:103]: To clarify, "...in both" - Does this mean the master pool needs 4 CPUs and 4 GB of memory and the work pool needs the same  - 4 CPUs and 4 GB of memory - for each node?

**Review by ritawatson — APPROVED** [review]
Approve with a small suggestion.

File: docs/docs-content/clusters/public-cloud/aws/create-cluster.md (hunk 9 above)
Comment (lennessyy) [reply docs/docs-content/clusters/public-cloud/aws/create-cluster.md:103]: The master pool needs for 4CPUs and 4GB across all nodes, and the worker pool needs 4CPUs and 4GB of memory across all nodes across all pools.

**Pull request merged**